app/helpers/month_names_helper.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+module MonthNamesHelper
+  def format_month_name(month_index)
```

Comment:
In Ruby we generally refer to what is returned by the method, not about what the method is doing. So a more "ruby"-like name would be "formatted_month_name". Of course, this is subjective. So what do you think about the idea?